### PR TITLE
Update footer ad format

### PR DESCRIPTION
**Pull request by sayzlim**
Remove the icon and style it to match the docs theme. Refer to the screenshot below for the new design:

### CODE REVIEW

**Comment by phanan** [comment]
Thanks for the PR @sayzlim!
@yyx990803 I don't see any problem, but it would be better if you take a look as well, I guess.

File: themes/vue/source/css/_ad.styl [hunk 1]
```diff
@@ -30,41 +30,37 @@
...
-        height 20px
-        border-radius 3px
-        vertical-align middle
-        position relative
-        top -1px
-    .default-title
+#bsa-native
+  .native-box
+    box-shadow inset 0 0 0 1px hsla(0, 0%, 0%, .075)
+    background-color: #f8f8f8;
```
Comment (phanan) [inline themes/vue/source/css/_ad.styl]: You can remove the colon in these rules btw.

Comment (sayzlim) [reply themes/vue/source/css/_ad.styl]: @phanan thanks for reviewing! I’ve removed the colon in these parts. Do we want to remove all the colons in this stylesheet?

Comment (phanan) [reply themes/vue/source/css/_ad.styl]: I'd say it would be nice for you to do so, as long as we don't break anything :D

Comment (sayzlim) [reply themes/vue/source/css/_ad.styl]: Please review one more time. Should be good! :)

**Pull request merged**